tensorrec/eval.py

```diff
@@ -59,6 +59,77 @@ def recall_at_k(model, test_interactions, k=10, user_features=None, item_feature
     return hit / retrieved
 
 
+def _idcg(hits, k=10, ctype="binary"):
+
+    if ctype == "binary":
+        arg = min(hits, k+1)
```

Comment:
What's the purpose of capping at k+1?
if I have
```
hits = np.array([3, 3, 3, 2, 2, 2, 1, 0])
k=2
```
what is the purpose of capping the relevance scores?